stack/rush-stack-compiler/src/Constants.ts

```diff
@@ -0,0 +1,60 @@
+// Copyright (c) Microsoft Corporation. All rights reserved. Licensed under the MIT license.
+// See LICENSE in the project root for license information.
+
+import * as path from 'path';
+
+const SRC_FOLDER_NAME: string = 'src';
+const LIB_FOLDER_NAME: string = 'lib';
+const DIST_FOLDER_NAME: string = 'dist';
+const TEMP_FOLDER_NAME: string = 'temp';
+
+/**
+ * @beta
+ */
+export class Constants {
```

Comment:
>Constants [](start = 13, length = 9)

"StandardBuildFolders"

Reply:
Long term these APIs should go in a **rush-stack-core** package.  Feel free to start the process to create the NPM package name

---
In reply to: [228676867](https://github.com/Microsoft/web-build-tools/pull/905#discussion_r228676867) [](ancestors = 228676867)